app/src/main/java/org/wikipedia/descriptions/DescriptionEditFragment.kt

```diff
@@ -487,6 +486,7 @@ class DescriptionEditFragment : Fragment() {
             when (action) {
                 DescriptionEditActivity.Action.ADD_DESCRIPTION -> {
                     if (binding.fragmentDescriptionEditView.wasSuggestionChosen) {
+                        tags.add(EditTags.APP_DESCRIPTION_ADD)
```

Comment:
Does a machine-generated description need its own edit tag? It used to have `#machine-suggestion` in the edit summary. 
https://phabricator.wikimedia.org/T366730#9924336

Reply:
I remember hearing in one of the data syncs that these hashtags do not need to be converted to tags. We can raise it again at some point, but that needn't block this PR.